src/shared/templates/shared/partial/popover_tour.html

```diff
@@ -0,0 +1,384 @@
+        <div class="embed-fluid embed-fluid-16x9 mb-0_5">
+            <iframe id="contextIframe" src="https://videos.sproutvideo.com/embed/799edbb41410e3c1f0/10056e5843bb7d55?type=hd" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen aria-label="Context tools video"></iframe>
+        </div>
+        <p>Select some text to see things you can do with it.</p>
+        <ul class="list feature-list">
+            <li><span class="feature-icon icon-edit" aria-hidden="true"></span> Highlight the passage</li>
+            <li><span class="feature-icon icon-book" aria-hidden="true"></span> Look up a definition</li>
+            <li><span class="feature-icon icon-switch" aria-hidden="true"></span> Translate or simplify</li>
+            <li><span class="feature-icon icon-play-circled2" aria-hidden="true"></span> Hear the words read aloud</li>
+        </ul>
+        <div class="row gx-0_5 flex-items-center">
+            <div class="col">
+                {% if has_teacher_resource %}
+                    {% if clusive_user.role == 'TE' or clusive_user.role == 'PA' %}
+                        <!-- 'context' type popovers do not have a "Learn more" resource link yet, so this is blank -->
+                        <a href="#" tabindex="-1"></a>
+                    {% endif %}
+                {% endif %}
+            </div>
+            <div class="col-auto tooltip-action">
+                <button type="button" class="btn btn-primary" data-cfw-dismiss="popover">Got it!</button>
+            </div>
+        </div>
+        {% elif tour_name == "view" %}
+<!-- view -->
+        <div class="embed-fluid embed-fluid-16x9 mb-0_5">
+            <iframe id="viewIframe" src="https://videos.sproutvideo.com/embed/ac9edbbb141fe6c525/86b0cbe36e2ab800?type=hd" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen aria-label="Library Views video"></iframe>
+        </div>
+        <p>View the Clusive library by all readings, or in smaller groupings like class readings, readings you’ve starred, public readings, or readings you’ve uploaded.</p>
+        <div class="row gx-0_5 flex-items-center">
+            <div class="col">
+                {% if has_teacher_resource %}
+                    {% if clusive_user.role == 'TE' or clusive_user.role == 'PA' %}
+                        <!-- 'view' type popovers currently do not have a "Learn more" resource link yet, so this is blank -->
+                        <a href="#" tabindex="-1"></a>
+                    {% endif %}
+                {% endif %}
+            </div>
+            <div class="col-auto">
+                <button type="button" class="btn btn-primary" data-cfw-dismiss="popover">Got it!</button>
+            </div>
+        </div>
+        {% elif tour_name == "book_actions" %}
+<!-- book_actions -->
+        {% if clusive_user.role == 'TE' %}
```

Comment:
Yeah, I'll see if I can refactor and simplify the template.  As for the missing "or PA", that's something like a copy-paste error.

Reply:
@klown Most of my refactoring in popover_tour.html was around the 'footer' section, as seen here -  you will need to click the 'load diff' link to see the changes: [#336 - popover_tour.html](https://github.com/cast-org/clusive/pull/336/files#diff-eb5c13490b9c5999f2faa4709aacae3eb06246c6dce38c67a9f011f8533a0d3a)

Reply:
Thanks @mbrambilla.  I thought about merging just your footer change into this PR, but it has features that don't apply here or might break things.  I'll work on making a suitable version suitable.  Since that is where most of your refactoring was done, it should make merge conflicts smaller.
🤞